### ISSUE TITLE
fast_strstr("barium iodide", "iodide"); returns null

### ISSUE THREAD

**Issue by alexbk66** [RaphaelJ/fast_strstr#8]

**Comment by kirabou** [comment]
Have you tried the change I suggested 3 years ago (see the pull request) ? It may solve your issue, but it has never been committed by the owner of this code.

**Comment by alexbk66** [comment]
Yeah, thank you, your suggestion worked. But unfortunately performance is worse than standard strstr...
